package.json

```diff
@@ -36,6 +37,7 @@
     "eslint-config-prettier": "^6.5.0",
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-react": "^7.16.0",
+    "fs-extra": "^9.0.1",
```

Comment:
New dependency?

Reply:
Yea. It's effectively the same module as the core fs module. Just promise-based.